test/global-decompress.test.js

```diff
@@ -18,8 +18,8 @@ function createPayload (compressor) {
   return payload
 }
 
-test('It should decompress the request payload :', async (t) => {
-  t.test('using brotli algorithm when `Content-Encoding` request header value is set to `br`', async (t) => {
+describe('It should decompress the request payload :', async () => {
+  test('using brotli algorithm when `Content-Encoding` request header value is set to `br`', async (t) => {
     t.plan(2)
```

Comment:
but node:test has no t.plan?

Reply:
https://nodejs.org/api/test.html#contextplancount ?

Reply:
TIL. You just need to use `t.assert.<someassertionFn>` to make it count.